releasenotes/notes/credential-name-support-egress-sidecar.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+releaseNotes:
+   - |
+     **Added** the ability to set credentialName based secret configuration
+     at sidecars for egress TLS traffic when WorkloadSelector is specified in `DestinationRule`
```

Comment:
Do we need to declare the sidecar should have the permission to list secrets in the namespace where it resides.

Reply:
Added